.eslintrc.cjs

```diff
@@ -1,73 +1,49 @@
 module.exports = {
-  parser        : '@typescript-eslint/parser',
-  parserOptions : {
-    ecmaVersion : 'latest', // Allows the use of modern ECMAScript features
-    sourceType  : 'module', // Allows for the use of imports
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 'latest', // Allows the use of modern ECMAScript features
+    sourceType: 'module', // Allows for the use of imports
   },
   plugins: [
     '@typescript-eslint',
-    'todo-plz' // for enforcing TODO formatting to require "github.com/TBD54566975/dwn-server/issues/"
+    'todo-plz', // for enforcing TODO formatting to require "github.com/TBD54566975/dwn-server/issues/"
   ],
   env: {
-    node    : true, // Enable Node.js global variables
-    browser : true
+    node: true, // Enable Node.js global variables
+    browser: true,
   },
   rules: {
-    'curly'      : ['error', 'all'],
-    'no-console' : 'off',
-    'indent'     : [
-      'error',
-      2
-    ],
-    'object-curly-spacing' : ['error', 'always'],
-    'linebreak-style'      : [
-      'error',
-      'unix'
-    ],
-    'quotes': [
-      'error',
-      'single',
-      { 'allowTemplateLiterals': true }
-    ],
-    '@typescript-eslint/semi' : ['error', 'always'],
-    'semi'                    : ['off'],
-    'no-multi-spaces'         : ['error'],
-    'no-trailing-spaces'      : ['error'],
-    'max-len'                 : ['error', { 'code': 150, 'ignoreStrings': true }],
-    'key-spacing'             : [
```

Comment:
removed as most of them covered by prettier, so separating formatting from code rules